run_tribler.py

```diff
@@ -115,16 +115,18 @@ def start_tribler():
             from TriblerGUI.tribler_window import TriblerWindow
 
             app = TriblerApplication("triblerapp", sys.argv)
-
+            if app.is_running():
+                for arg in sys.argv[1:]:
+                    if os.path.exists(arg) and arg.endswith(".torrent"):
+                        app.send_message("file:%s" % arg)
```

Comment:
Maybe we should add a test for this? If it is not very hard to do, of course.

Reply:
ok